src/app/retrospective-data-entry/components/banner/retrospective-banner.component.html

```diff
@@ -1,6 +1,14 @@
 <div class="retrospective-banner" *ngIf="retroEnabled">
   <retrospective-data-entry-settings (onSettingsChange)="toggleDialog($event)" [bannerMode]="true"></retrospective-data-entry-settings>
-  <p-dialog header="Retrospective Data Entry" [(visible)]="displayDialog" minHieght="300" position="center" modal="true">
-    <retrospective-data-entry-settings (onSettingsChange)="toggleDialog($event)" [modalMode]="true" [bannerMode]="false"></retrospective-data-entry-settings>
-  </p-dialog>
+  <div class="dialog-wrapper" *ngIf="displayDialog">
+    <div class="dialog-wrapper-content">
+      <p-dialog header="Retrospective Data Entry" 
+      [(visible)]="displayDialog" 
+      minHieght="300" 
```

Comment:
Should this be `minHeight`?

Reply:
I have removed it. Didn't seem to have any effect on the p-dialog. min height for later versions is passed via styles props

Reply:
Nice. lgtm